frontend/packages/core/src/AppProvider/workflow.tsx

```diff
@@ -42,6 +42,12 @@ interface Route {
   requiredConfigProps?: string[];
   /** Is the workflow discoverable via search and drawer navigation. This defaults to false. */
   hideNav?: boolean;
+  /**
+   * The feature flag used to determine if the route should be registered.
+   *
+   * If this is not set the route will always be registered.
+   */
+  featureFlag?: string;
```

Comment:
can we derive the flag from existing uniquely identifiable information about the workflow instead of requiring the user to provide one?

this makes it impossible for us to publish a canonical feature flag for a workflow.

Reply:
We could use something like the package name and route key but I was thinking it might be nice to allow users to specify a flag if they want to be able to enable all routes behind a single flag